app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -408,6 +408,63 @@ def test_update_experiment_is_rollout(self):
         experiment = NimbusExperiment.objects.get(id=experiment_id)
         self.assertTrue(experiment.is_rollout)
 
+    def test_update_live_rollout(self):
+        user_email = "user@example.com"
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LAUNCH_APPROVE_APPROVE,
+            application=NimbusExperiment.Application.FENIX,
+            is_rollout=True,
+            population_percent=40,
+        )
+        experiment_id = experiment.id
+        response = self.query(
+            UPDATE_EXPERIMENT_MUTATION,
+            variables={
+                "input": {
+                    "id": experiment.id,
+                    "populationPercent": "50",
+                    "changelogMessage": "test changelog message",
+                }
+            },
+            headers={settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+        self.assertEqual(response.status_code, 200)
+
+        experiment = NimbusExperiment.objects.get(id=experiment_id)
+        self.assertTrue(experiment.is_rollout)
+        self.assertEqual(experiment.population_percent, 50.0)
+        self.assertEqual(experiment.publish_status, NimbusConstants.PublishStatus.DIRTY)
+        self.assertEqual(experiment.status, NimbusConstants.Status.LIVE)
+        self.assertEqual(experiment.status_next, None)
+
+    def test_do_not_update_live_experiment(self):
+        user_email = "user@example.com"
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LAUNCH_APPROVE_APPROVE,
+            application=NimbusExperiment.Application.FENIX,
+            is_rollout=False,
+            population_percent=40,
+        )
+        experiment_id = experiment.id
+        response = self.query(
+            UPDATE_EXPERIMENT_MUTATION,
+            variables={
+                "input": {
+                    "id": experiment.id,
+                    "populationPercent": "50",
+                    "changelogMessage": "test changelog message",
+                }
+            },
+            headers={settings.OPENIDC_EMAIL_HEADER: user_email},
+        )
+        self.assertEqual(response.status_code, 200)
```

Comment:
I assume we expect some kind of error to be here in the response, I'd expect to check that here in the test.

Reply:
I agree it should return or raise some error, instead of 200 status 🤔

Reply:
Oh well no I mean graphql will always return a 200, sadly, but there will be errors in the response body we should probably assert on here.

Reply:
yeah right, I mean review errors can be used here 😆